package.json

```diff
@@ -65,12 +65,13 @@
         "@jupyterlab/rendermime": "^4.4.0",
         "@jupyterlab/settingregistry": "^4.4.0",
         "@jupyterlab/ui-components": "^4.4.0",
-        "@langchain/anthropic": "^0.3.9",
-        "@langchain/community": "^0.3.44",
-        "@langchain/core": "^0.3.57",
-        "@langchain/mistralai": "^0.1.1",
-        "@langchain/ollama": "^0.2.0",
-        "@langchain/openai": "^0.4.4",
+        "@langchain/anthropic": "^0.3.22",
+        "@langchain/community": "^0.3.46",
+        "@langchain/core": "^0.3.58",
+        "@langchain/google-genai": "^0.2.12",
```

Comment:
The latest version of `@langchain/google-genai` seems to depend on newer releases of `@langchain/core`, so bumping the other langchain dependencies here.